src/calculateRentalCost.js

```diff
@@ -4,7 +4,44 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const discountUpToSix = 20;
+  const discountSevenMore = 50;
+
+  if (days <= 0) {
+    return 'You need to rent for at least one day';
+  }
+
+  if (days <= 2) {
+    return days * pricePerDay;
+  } else if (days >= 3 && days < 7) {
+    return days * pricePerDay - discountUpToSix;
```

Comment:
The condition `days >= 3 && days < 7` can be simplified to `days < 7` since the previous return statement already ensures that `days` is greater than 2.